spec/features/user_edits_account_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+feature 'user edits account', %Q{
+
+  As a registered user
+  I want to be able to edit my account
+  So my information is up to date
+  } do
+    #Acceptance Criteria
+    # *I must be logged in
+    # *I can only edit my own account
+    # *I can change or upload an optional avatar
+    # *I can change my password
+    # *I can change my email
+    # *I cannot change my username
+
+    scenario 'authenticated user changes email, password, & avatar' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.